### ISSUE TITLE
Image copyrights

### ISSUE THREAD

**Issue by kurgol** [Kurgol/samba-action-calculator#1]
The "blue dice" background image copyrights Elyoukey used for the website cannot be tracked down, we should replace them with images we can show copyright for.

I may approach the https://fumbbl.com/ guys about using renders of their custom block dice.

**Comment by kurgol** [comment]
I have sent a message to "whatball" the creator of the fumbbl dice designs and requested permission to replace all dice/block dice images with the fumbbl designs.

**Comment by chris-shortall** [comment]
I'd be happy to create some images (background / block dice symbols / app icon) as a fall-back option, but would be cool to use the fumbbl dice.

**Comment by kurgol** [comment]
Indeed. It would be nice if selecting your block dice appearance was a feature :) I may even contact the community managers at cyanide, they may be willing to allow some reuse of their artwork.
